composer.json

```diff
@@ -355,7 +355,7 @@
         ],
         "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
         "nuke": [
-            "rm -rf vendor composer.lock docroot/core docroot/modules/contrib docroot/profiles/contrib docroot/themes/contrib",
+            "rm -rf vendor docroot/core docroot/modules/contrib docroot/profiles/contrib docroot/themes/contrib",
```

Comment:
Oh I also included this because I was using `composer nuke` quite a bit. Turned out to be unrelated to this PR but I think removing `composer.lock` would make this more useful.

Reply:
Baha you read my mind. I was figuring out where that came from. I've had to run that a bit as well so makes sense to me 🤷🏼‍♀️